tests/components/hisense_aehw4a1/test_init.py

```diff
@@ -0,0 +1,87 @@
+"""Tests for the Hisense AEH-W4A1 init file."""
+from unittest.mock import patch
+
+from homeassistant import config_entries, data_entry_flow
+from homeassistant.components import hisense_aehw4a1
+from homeassistant.setup import async_setup_component
+
+from tests.common import mock_coro
+
+
+async def test_creating_entry_sets_up_climate_discovery(hass):
+    """Test setting up Hisense AEH-W4A1 loads the climate component."""
+    with patch(
+        "homeassistant.components.hisense_aehw4a1.config_flow.AehW4a1.discovery",
+        return_value=mock_coro(["1.2.3.4"]),
```

Comment:
`discovery` is a normal function, right? Don't use `mock_coro` in the return value.